### ISSUE TITLE
Noisebridge Announce Moderation Improvements

### ISSUE THREAD

**Issue by ruthgrace** [noisebridge/infrastructure#127]
Anybody know if we can set up noisebridge-announce email approvals via Slack? Such that anyone can approve, but also you can see who's approving what and stop somebody if they're approving spam?

I just approved my own email about the laser cutter being fixed, and there were a few emails in there about events that have already passed. I approved them all because they weren't spam, but it would be nice of they had been approved earlier

**Comment by nthmost** [comment]
Has someone been approving spam?

**Comment by ruthgrace** [comment]
No, there is spam that is appropriately rejected. The issue is that there's not enough approval generally. So I'd like to make messages easier to approve while having accountability to prevent bad approvals.

**Comment by nthmost** [comment]
Why do we need accountability to prevent bad approvals?

**Comment by ruthgrace** [comment]
I think I'm being unclear: I want to have announce mailing list approvals
available as something you can do on slack just like social media tool,
where anyone can do it.

**Comment by marcidy** [comment]
@ruthgrace There's an API available which could be integrated like SMT. I am not sure if this is compatible with the version we are running, but it is with mailman3, which we are planning to upgrade at ...somepoint :D.

http://docs.mailman3.org/projects/mailmanclient/en/latest/index.html

I dont feel like dealing with another noisebridge API thing, im behind on all the ones I started.  It would be great if someone stepped up to help.  Happy to guide them.
